lib/middleware/redirects.js

```diff
@@ -30,14 +30,22 @@ function addQuery(url, qs) {
   return url;
 }
 
-var Redirect = function(source, destination, type) {
+var Redirect = function(glob, pattern, destination, type) {
```

Comment:
Let's call this `regex` not `pattern` since that's consistent with the API and config file.

Reply:
Done.